application/testing/CMakeLists.txt

```diff
@@ -846,6 +846,9 @@ f3d_test(NAME TestInteractionNonExistentRecordDirectory ARGS --interaction-test-
 f3d_test(NAME TestInteractionRecord DATA cow.vtp ARGS --interaction-test-record=${CMAKE_BINARY_DIR}/Testing/Temporary/interaction.log NO_BASELINE)
 f3d_test(NAME TestInteractionPlay DATA cow.vtp ARGS --interaction-test-play=${CMAKE_BINARY_DIR}/Testing/Temporary/interaction.log DEPENDS TestInteractionRecord NO_BASELINE)
 
+# Command Script Test
+f3d_test(NAME TestCommandScriptMissingFile DATA dragon.vtu ARGS --command-script=missing_script.txt REGEXP "Unable to open command script file" NO_BASELINE)
```

Comment:
nice, please add an actual test using a script that you can put in `testing/data`